release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -988,6 +1138,17 @@ Support for running {op-system-base-full} 7 compute machines in {product-title}
 
 You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. You must deploy new {op-system-base} 8 hosts, and the old {op-system-base} 7 hosts must be removed.
 
+[id="ocp-4-10-third-party-monitoring-component-uis-removal"]
+==== Third-party monitoring component user interface access removed
+
+With this release, you can no longer access third-party web user interfaces (UIs) for the following monitoring stack components:
+
+* Alertmanager
+* Thanos Querier
+
```

Comment:
if enabled user workload monitoring, Thanos Ruler UI is also unavailable

Reply:
Added `Thanos Ruler (if user workload monitoring is enabled)` to the list